### PR TITLE
Clarify authors.

### PR DESCRIPTION
**Pull request by vext01**
The main change is 3fbad79, then there's a commit to make the README overview better.

### CODE REVIEW

File: ykview/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ykview"
 version = "0.1.0"
-authors = ["bjorn3 <bjorn3@users.noreply.github.com>"]
+authors = ["Edd Barrett <vext01@gmail.com>"]
```
Comment (bjorn3) [inline ykview/Cargo.toml:4]: ???

```suggestion
authors = ["The Yorick Developers"]
```

Comment (vext01) [reply ykview/Cargo.toml:4]: Oops! That was a copy-paste error! I assure you I wasn't trying to take credit!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+from Rust. Imagine PyPy, but where your interpreter is written in Rust instead
+of RPython.
+
+This repository houses the non-compiler components of the system.
+
+The compiler repo can be found [here](https://github.com/softdevteam/ykrustc).
+
+## Contributors
+
+Yorick is developed by the Software Development Team at King's College London,
```
Comment (ltratt) [inline README.md:13]: I think we can be more inclusive here. Maybe something like "Yorick is developed by a team of people which can be seen in the <graph of contributors>." I don't think we need a special mention.

Comment (vext01) [reply README.md]: Fixed in 12d05fc4eb8d7f82ba13816fca869faab8752963

**Review by bjorn3 — APPROVED** [review]

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Comment by vext01** [comment]
bors try

**Comment by ltratt** [comment]
`cargo` not being found is a bit odd! Let's try again.

bors r+

**Comment by vext01** [comment]
I wonder if the upstream sync broke it. Maybe cargo is installed in a
different location or something.

I'll look into it when I find a moment.

**Comment by vext01** [comment]
Made a little progress with this. Something seems to have changed with `x.py install` such that the directory we tar up for the CI snapshot is bogus, and only contains:
```
$ find build/ykrustc-stage2-latest/
build/ykrustc-stage2-latest/
build/ykrustc-stage2-latest/libexec
build/ykrustc-stage2-latest/libexec/cargo-credential-1password
build/ykrustc-stage2-latest/VERSION
```

I also notice something wrong with the CI snapshot directory:
```
-rwxrwxr-x  1 buildbot-worker1 buildbot-workers 124643091 Dec 15 17:18 ykrustc-hw-stage2-1b4b37bf.tar.bz2
-rwxrwxr-x  1 buildbot-worker3 buildbot-workers 124650393 Dec 15 17:26 ykrustc-hw-stage2-5ccb675d.tar.bz2
-rwxrwxr-x  1 buildbot-worker4 buildbot-workers 124634002 Dec 15 17:23 ykrustc-hw-stage2-60d5035b.tar.bz2
-rwxrwxr-x  1 buildbot-worker4 buildbot-workers    298379 Dec 21 13:48 ykrustc-hw-stage2-92cb5763.tar.bz2
-rwxrwxr-x  1 buildbot-worker1 buildbot-workers 124638644 Dec 21 13:10 ykrustc-hw-stage2-b7178dda.tar.bz2
-rwxrwxr-x  1 buildbot-worker2 buildbot-workers 124786567 Dec 17 14:59 ykrustc-hw-stage2-ec7fe7f5.tar.bz2
lrwxrwxrwx  1 buildbot-worker4 buildbot-workers        61 Dec 21 13:48 ykrustc-hw-stage2-latest.tar.bz2 -> /opt/ykrustc-bin-snapshots/ykrustc-hw-stage2-92cb5763.tar.bz2
-rwxrwxr-x  1 buildbot-worker2 buildbot-workers 124668885 Dec 16 07:21 ykrustc--stage2-5ccb675d.tar.bz2
-rwxrwxr-x  1 buildbot-worker3 buildbot-workers    298217 Dec 23 07:21 ykrustc--stage2-92cb5763.tar.bz2
lrwxrwxrwx  1 buildbot-worker3 buildbot-workers        59 Dec 23 07:21 ykrustc--stage2-latest.tar.bz2 -> /opt/ykrustc-bin-snapshots/ykrustc--stage2-92cb5763.tar.bz2
```

My guess there is that the removal of `STD_TRACER_MODE` wasn't done quite right.

Out of time for now :)

**Comment by ltratt** [comment]
Aha, OK!

**Comment by vext01** [comment]
I've been slowly investigating this. It seems the installer is currently in flux, and this is likely the source of our problems: https://rust-lang.zulipchat.com/#narrow/stream/182449-t-compiler.2Fhelp/topic/.60x.2Epy.20install.60.20broken.3F

**Comment by ltratt** [comment]
It looks like there's a PR fix up now? https://github.com/rust-lang/rust/pull/80514

**Comment by vext01** [comment]
Once that's merged, I'll sync upstream and hopefully it will fix our
issues. We actually saw a different problem, but I'm hoping it's related.

I'm also going to add CI steps that check that certain binaries exist in
the install dir, like cargo and rustc. Then we can catch this kind of thing
earlier next time.

**Comment by ltratt** [comment]
I've checked and https://github.com/rust-lang/rust/pull/80514 has been merged into rust's master branch, even though the PR is still open. So I think we're good to go!

**Comment by vext01** [comment]
Using this morning's upstream master, `x.py install` no longer fails, but it doesn't actually put anything in the install dir.

**Comment by ltratt** [comment]
Ye gods :/

**Comment by vext01** [comment]
bors try

**Comment by vext01** [comment]
@ltratt this is good to go.

**Comment by ltratt** [comment]
Do you want/need to rebase against master?

**Comment by vext01** [comment]
It's already up to date, so we can go ahead.

**Pull request merged**